### PR TITLE
add three windings transformers for expert filters

### PR DESCRIPTION
**Pull request by EtienneLt**

### CODE REVIEW

File: src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java [hunk 1]
```diff
@@ -266,9 +295,11 @@ private static String getRatioTapChangerFieldValue(FieldType field, @Nullable Ra
...
         }
         return switch (field) {
-            case RATIO_TARGET_V -> String.valueOf(ratioTapChanger.getTargetV());
-            case LOAD_TAP_CHANGING_CAPABILITIES -> String.valueOf(ratioTapChanger.hasLoadTapChangingCapabilities());
-            case RATIO_REGULATION_MODE -> String.valueOf(getRatioRegulationMode(ratioTapChanger));
+            case RATIO_TARGET_V, RATIO_TARGET_V1, RATIO_TARGET_V2, RATIO_TARGET_V3 -> String.valueOf(ratioTapChanger.getTargetV());
+            case LOAD_TAP_CHANGING_CAPABILITIES, LOAD_TAP_CHANGING_CAPABILITIES_1, LOAD_TAP_CHANGING_CAPABILITIES_2,
+                 LOAD_TAP_CHANGING_CAPABILITIES_3 -> String.valueOf(ratioTapChanger.hasLoadTapChangingCapabilities());
+            case RATIO_REGULATION_MODE, RATIO_REGULATION_MODE_1, RATIO_REGULATION_MODE_2, RATIO_REGULATION_MODE_3
+                -> String.valueOf(getRatioRegulationMode(ratioTapChanger));
```
Comment (thangqp) [inline src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java:302]: In my opinion, it is better one line for each FieldType matched for visibility reason.

File: src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java [hunk 2]
```diff
@@ -292,8 +323,8 @@ private static String getPhaseTapChangerFieldValue(FieldType field, @Nullable Ph
             return null;
         }
         return switch (field) {
-            case PHASE_REGULATION_VALUE -> String.valueOf(phaseTapChanger.getRegulationValue());
-            case PHASE_REGULATION_MODE -> String.valueOf(getPhaseRegulationMode(phaseTapChanger));
+            case PHASE_REGULATION_VALUE, PHASE_REGULATION_VALUE_1, PHASE_REGULATION_VALUE_2, PHASE_REGULATION_VALUE_3 -> String.valueOf(phaseTapChanger.getRegulationValue());
+            case PHASE_REGULATION_MODE, PHASE_REGULATION_MODE_1, PHASE_REGULATION_MODE_2, PHASE_REGULATION_MODE_3 -> String.valueOf(getPhaseRegulationMode(phaseTapChanger));
```
Comment (thangqp) [inline src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java:327]: Idem, one line for each FieldType

File: src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java [hunk 3]
```diff
@@ -329,6 +360,59 @@ private static String getTwoWindingsTransformerFieldValue(FieldType field, Strin
...
+                 SERIE_REACTANCE_3, MAGNETIZING_CONDUCTANCE_3, MAGNETIZING_SUSCEPTANCE_3, HAS_RATIO_TAP_CHANGER_3,
+                 RATIO_TARGET_V3, LOAD_TAP_CHANGING_CAPABILITIES_3, RATIO_REGULATION_MODE_3, HAS_PHASE_TAP_CHANGER_3,
+                 PHASE_REGULATION_MODE_3, PHASE_REGULATION_VALUE_3, SUBSTATION_PROPERTIES_3, VOLTAGE_LEVEL_PROPERTIES_3
+                -> getTwtLegFieldValue(field, propertyName, threeWindingsTransformer.getLeg3());
+            default ->
+                throw new PowsyblException(FIELD_AND_TYPE_NOT_IMPLEMENTED + " [" + field + "," + threeWindingsTransformer.getType() + "]");
+        };
+    }
+
+    private static String getTwtLegFieldValue(FieldType field, String propertyName, ThreeWindingsTransformer.Leg leg) {
```
Comment (thangqp) [inline src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java:387]: Idem, one lien for each matched FieldType

File: src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java (hunk 1 above)
Comment (EtienneLt) [reply src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java:302]: do I repeat the line like 
```java
case RATIO_TARGET_V -> String.valueOf(ratioTapChanger.getTargetV());
case RATIO_TARGET_V1 -> String.valueOf(ratioTapChanger.getTargetV());
```
or do I skip line for each enum?
```java
case RATIO_TARGET_V, 
         RATIO_TARGET_V1, 
         RATIO_TARGET_V2 -> String.valueOf(ratioTapChanger.getTargetV());
```

**Review by thangqp — APPROVED** [review]

**Pull request merged**